app/forms/declaration/PackageInformation.scala

```diff
@@ -91,7 +94,7 @@ object PackageInformation extends DeclarationPage {
     .verifying("supplementary.packageInformation.empty", validatePackageInformation(_))
 
   private def validatePackageInformation(packageInformation: PackageInformation): Boolean =
-    !(packageInformation.typesOfPackages.isEmpty && packageInformation.numberOfPackages.isEmpty && packageInformation.shippingMarks.isEmpty)
+    !(packageInformation.isEmpty)
```

Comment:
I think this is not required to be in the method.

Brackets around `packageInformation.isEmpty` are not necessary

Reply:
They're not, but they make the `!` more visible

Reply:
Ahh.... now I understand. Let me amend this little change.